repositories.bzl

```diff
@@ -277,6 +280,15 @@ def io_opencensus_grpc_metrics():
       sha1 = "d57b877f1a28a613452d45e35c7faae5af585258",
   )
 
+def javax_annotation():
+  # TODO(davido): maven_jar does not support neverlink attribute.
+  # To circumvent use http_file following by java_import.
+  native.http_file(
+      name = "javax_annotation_api",
+      sha256 = "e04ba5195bcd555dc95650f7cc614d151e4bcd52d29a10b8aa2197f3ab89ab9b",
+      urls = ["http://central.maven.org/maven2/javax/annotation/javax.annotation-api/1.3.2/javax.annotation-api-1.3.2.jar"],
```

Comment:
We use version 1.2 in build.gradle and like to keep the versions in sync. 1.3.2 also doesn't support Java 7 (although I honestly don't know if Bazel supports Java 7). Let's change this to version 1.2 (unless there's some problem with doing so).

Reply:
Done.